src/ucp/core/ucp_ep.c

```diff
@@ -97,6 +97,11 @@ static uct_ep_t ucp_failed_tl_ep = {
 };
 
 
+int ucp_ep_is_uct_ep_failed(uct_ep_h uct_ep)
```

Comment:
then just `ucp_is_uct_ep_failed`, since actually there is no ucp ep at all

Reply:
fixed